packages/ui/src/components/Tabs/STab.vue

```diff
@@ -0,0 +1,143 @@
+<script setup lang="ts">
+
+import { ref } from 'vue';
+
+
+const props = withDefaults(
+  defineProps<{
+    disabled?: boolean,
+    borderRadius?: string,
+  }>(), {
+  disabled: false,
+  borderRadius: ''
+}
+)
+
+const instance = getCurrentInstance();
+
+
+const {active, tabs, selectTab}  = inject("tabsState", {

```

Comment:
- Use injection via Symbol. Create a separated variable for injection key and export it from the library too
- Use typed injection, e.g. `inject<InjectData>(...)`, or better:
   ```ts
   type InjectData = { /* ... */ }
   const key: InjectionKey<InjectData> = Symbol('TabsState')
   const { /* ... */ } = inject(key) // auto inferred
   ```